app/controllers/katello/api/v2/content_view_versions_controller.rb

```diff
@@ -25,7 +25,7 @@ class Api::V2::ContentViewVersionsController < Api::V2::ApiController
     def index
       options = {
         :includes => [:content_view, :environments, :composite_content_views, :history => :task],
-        :custom_sort => lambda { |query| query.order("#{ContentViewVersion.table_name}.major desc, #{ContentViewVersion.table_name}.minor desc") }
+        :custom_sort => sort_by_version
       }
```

Comment:
My recent changed switched us to use scoped_search based sorting, so i think just passing in 'version' and 'desc' instead of 'nil, nil' below would work (since version is a searchable field)

Reply:
I'll try it, but if you pass params[:order] as 'version DESC', you get an error that it's not a valid column - and that originates from the scoped_search method

Reply:
@stbenjam are you sure your on latest master?

Reply:
What commit fixed this? This was not working when I started yesterday :disappointed: 

I don't think my PR is needed at all then and I can mark it as a dupe of yours.